impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/passivating/PassivatingContextTest.java

```diff
@@ -49,6 +50,7 @@
  * @author Martin Kouba
  */
 @SpecVersion(spec = "cdi", version = "2.0")
+@Test(groups = CDI_FULL)
```

Comment:
Sometimes, we add `(groups = CDI_FULL)` to an existing `@Test` annotation on a method, and sometimes we add a brand new `@Test(groups = CDI_FULL)` annotation on a class. Is there a difference? (Honest question, I don't really know :-) )

Reply:
If there are multimple tests in a class I'm applying the group to, then I add it to the class level to avoid having to repeat it.

Reply:
Gotcha, thanks!